### DISCUSSION TITLE
Web App Certificate - Zmiana Key Vaulta na inny ?

### DISCUSSION THREAD

**Discussion by Mski89** [AzureCommunityPL/group#182] [Pomoc]
Cześć, 
W wyniku zmiany w infrstrukturze musze przeniesc certyfikat do innego KV. Jest jakiś sposób aby go przenieść bez usuwania  i tworzenia go od nowa?

Nie moge znalezc info na ten temat. A kombinowanie nie przyniosło żadnego rezultatu.

**Comment by kamil-mrzyglod** [comment]
Jaki masz dokładnie tutaj problem? Wchodzisz do swojego bazowego KV, klikasz na Download PFX/PEM a następnie z tym plikiem idziesz do nowego KV i klikasz Generate / Import. Nie widzę innej opcji 😉

**Reply by Mski89** [reply]
Z Web App Certifiate nie da się tak łatwo :) 
Nawet po usunieciu go z KV, po odświeżeniu pojawia się na nowo ponieważ jest przypisany do konkretnego KV i w tym właśnie jest problem, nie idzie tego zmienić.

**Reply by kamil-mrzyglod** [reply]
Pardon, nie doczytałem, że mówimy o zarządzalnym cercie :)
